x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/documentation/index.tsx

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import {
+  EuiAccordion,
+  EuiBasicTable,
+  EuiCode,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiSpacer,
+  EuiText,
+  EuiLink,
+  EuiBetaBadge,
+} from '@elastic/eui';
+import type { EuiInMemoryTableProps } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import type {
+  PackageInfo,
+  RegistryVarsEntry,
+  RegistryStream,
+  RegistryInput,
+} from '../../../../../types';
+import { useStartServices } from '../../../../../../../hooks';
+import { getStreamsForInputType } from '../../../../../../../../common/services';
+
+interface Props {
+  packageInfo: PackageInfo;
+  integration?: string | null;
+}
+
+export const DocumentationPage: React.FunctionComponent<Props> = ({ packageInfo, integration }) => {
+  const { docLinks } = useStartServices();
+
+  const content = (
+    <>
+      <EuiFlexGroup gutterSize="m" justifyContent="spaceBetween">
+        <EuiFlexItem grow={6}>
+          <EuiText>
+            <FormattedMessage
+              id="xpack.fleet.epm.packageDetails.apiReference.description"
+              defaultMessage="This document all the inputs, streams and variables available to use programmatically that integration via the Fleet Kibana API. {learnMore}"
```

Comment:
nit: `This document **shows** all the inputs, ... to use that integration programmatically via ...`

Reply:
Alternatively you could use:

```suggestion
              defaultMessage="This documents all the inputs, streams, and variables available to use this integration programmatically via the Fleet Kibana API. {learnMore}"
```